client/extensions/woocommerce/lib/order-taxes/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { get, reduce } from 'lodash';
+
+/**
+ * Get the total tax for the discount value
+ *
+ * @param {Object} order An order as returned from API
+ * @return {Float} Tax amount as a decimal number
+ */
+export function getOrderDiscountTax( order ) {
+	const tax = get( order, 'coupon_lines[0].discount_tax', 0 );
```

Comment:
I think this is at least part of the problem - since an order can have multiple coupons (see https://github.com/Automattic/wp-calypso/pull/17406#issuecomment-325044040 )